### PR TITLE
Remove unused imports #5100

### LINKED ISSUE #5100: Remove unused imports
There are a few warnings in eclipse showing unused imports. 
Such as in InstructorCoursesPageUiTest.java, org.openqa.selenium.By is never used.

### PR DESCRIPTION
**Pull request by yasirunilan**
Fixes #5100

### CODE REVIEW

**Comment by kanghj** [comment]
@yasirunilan, there's no need to make a new pr every time you have new changes, you can continue pushing code on the same branch in the other pr.

**Comment by yasirunilan** [comment]
So what do I need to do now

**Comment by thyageshm** [comment]
@yasirunilan if you push your new changes to the same branch as the one you created the PR from, the PR automatically reflects the new changes.

**Comment by thyageshm** [comment]
Choose one of the two PRs to continue and close the other one

**Comment by yasirunilan** [comment]
I closed one pull request now only one is remaining

**Comment by yasirunilan** [comment]
Kindly review

File: src/main/webapp/WEB-INF/appengine-web.template.xml [hunk 1]
```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (junhaoyap) [inline src/main/webapp/WEB-INF/appengine-web.template.xml]: This doesn't seem to have anything to do with unused import

File: src/test/resources/test.template.properties [hunk 2]
```diff
@@ -1,64 +0,0 @@
-#-----------------------------------------------------------------------------
```
Comment (junhaoyap) [inline src/test/resources/test.template.properties]: Same here

File: .settings/com.google.gdt.eclipse.core.template.prefs [hunk 3]
```diff
@@ -1,4 +0,0 @@
-eclipse.preferences.version=1
```
Comment (junhaoyap) [inline .settings/com.google.gdt.eclipse.core.template.prefs]: Same here

Comment (yasirunilan) [inline .settings/com.google.gdt.eclipse.core.template.prefs]: Yeah due to some miss configurations of the project,I had to change these files too. That's why I changed these files.It has nothing to do with unused imports.

**Comment by yasirunilan** [comment]
Yeah due to some miss configurations of the project,I had to change these files too. That's why I changed these files.It has nothing to do with unused imports.
 Add a line note

**Comment by junhaoyap** [comment]
Those are files needed for the project - they should not be removed

**Comment by yasirunilan** [comment]
I did copy the content from a original file to remove my miss configured content.So I think there would be no errors.

**Comment by damithc** [comment]
@yasirunilan the PR should contain only changes relevant to this PR. Your PR contains irrelevant changes (e.g. deleting of files) which means we cannot merge this PR until you get rid of them. If you cannot figure out how to get rid of them from this PR, close this PR and submit a fresh PR containing only the relevant changes.

**Comment by yasirunilan** [comment]
Yeah i think that's better.I'll do so.

**Comment by junhaoyap** [comment]
I will be setting this to be discarded then - the irrelevant changes should not be done when doing any PRs as this will affect other developers who rely on those files

**Pull request closed without merging**